src/main/java/org/launchcode/techjobs/oo/JobField.java

```diff
@@ -0,0 +1,36 @@
+package org.launchcode.techjobs.oo;
+
+import java.util.Objects;
+
+public abstract class JobField {
+    private int id;
+    private static int nextId = 1;
+    private String value;
+
+    public JobField() {
+        id = nextId;
+        nextId++;
+    }
+    public JobField(String aValue){
+        this();
+        this.value = aValue;
+    }
+    public int getId() {
+        return id;
+    }
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+    @Override
+    public String toString() {
+        return value;
+    }
+    @Override
+    public int hashCode() {
+        return Objects.hash(getId());
```

Comment:
```suggestion
        return Objects.hash(id);
```

You can also access the job field's `id` by the `id` itself instead of calling `getId()`.